### ISSUE TITLE
FetchError getaddrinfo EAI_AGAIN registry.npmjs.org

### ISSUE THREAD

**Issue by milahu** [raineorshine/npm-check-updates#1030]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 12`

---------------------------

after updating 757 packages, ncu throws

```
Upgrading /tmp/shared-editing-demo/node_modules/@babel/preset-env/package.json
[====================] 77/78 98%/tmp/.npm-global/pnpm-global/5/node_modules/.pnpm/npm-check-updates@12.1.0/node_modules/npm-check-updates/build/src/index.js:26
    throw err;
    ^

FetchError: request to https://registry.npmjs.org/@babel%2fplugin-proposal-logical-assignment-operators failed, reason: getaddrinfo EAI_AGAIN registry.npmjs.org
    at ClientRequest.<anonymous> (/tmp/.npm-global/pnpm-global/5/node_modules/.pnpm/minipass-fetch@1.4.1/node_modules/minipass-fetch/lib/index.js:110:14)
    at ClientRequest.emit (node:events:390:28)
    at TLSSocket.socketErrorListener (node:_http_client:447:9)
    at TLSSocket.emit (node:events:402:35)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  code: 'EAI_AGAIN',
  errno: 'EAI_AGAIN',
  syscall: 'getaddrinfo',
  hostname: 'registry.npmjs.org',
  type: 'system'
}
```

why 757 packages? probably random.

```
find . -name package.json -mmin -24 | wc -l # last package was installed 25 minutes ago
757
```

## Steps to Reproduce

```
cd $(mktemp -d)
git clone https://github.com/ipfs-labs/shared-editing-demo --depth 1
cd shared-editing-demo
npm install
cd node_modules
ncu -u --deep
```

## Current Behavior

throws

## Expected Behavior

retry fetch

**Comment by raineorshine** [comment]
npm-check-updates does not currently have retry functionality, but it could be useful.

**Comment by raineorshine** [comment]
Retry functionality added in #1033 and published to `v12.2.0`.

Defaults to 3 retries, which can be changed with `--retry <n>`.
